StyleCop.Analyzers/StyleCop.Analyzers/SpacingRules/SA1024ColonsMustBeSpacedCorrectly.cs

```diff
@@ -42,6 +41,12 @@ namespace StyleCop.Analyzers.SpacingRules
     /// }
     /// </code>
     ///
+    /// A colon that appears as part of format string component must not have leading whitespace characters. For example:
```

Comment:
I would prefer `string interpolation formatting component` instead of `format string component`.